reana_helloworld/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM condaforge/mambaforge:latest
+LABEL io.github.snakemake.containerized="true"
+LABEL io.github.snakemake.conda_env_hash="a27eb1bd907da7d709dcbe093527f3ae2ce3ac8b4a9f4ec63cc1a34b4e1f4dc0"
```

Comment:
I'm not actually sure what these lablels do or if they're necessary. So maybe we can either add comments that explain them, or just remove them.